lib/manageiq/style/cli.rb

```diff
@@ -121,6 +122,105 @@ def update_codeclimate_yml(file = ".codeclimate.yml")
         File.write(".codeclimate.yml", data.to_yaml.sub("---\n", ""))
       end
 
+      def update_gem_source
+        if (gemspec = Dir.glob("*.gemspec").first)
+          update_gemspec(gemspec)
+        elsif File.exist?("Gemfile")
+          update_gemfile
+        end
+      end
+
+      def update_gemspec(gemspec)
+        contents = File.read(gemspec)
+        return if contents.include?("manageiq-style")
+
+        lines = contents.lines
+
```

Comment:
How many of these empty lines did you add just to get an even +100 line diff? :trollface: 💯 
![image](https://user-images.githubusercontent.com/651659/105425678-d4da1f80-5c17-11eb-99d8-e056c5541231.png)

Reply:
Wow... nailed it 🥇